tests/10-rdfc-create.js

```diff
@@ -5,18 +5,17 @@
 import {
   shouldBeBs58, shouldBeMulticodecEncoded, verificationSuccess
 } from './assertions.js';
-import chai from 'chai';
 import {createInitialVc} from './helpers.js';
 import {documentLoader} from './documentLoader.js';
 import {endpoints} from 'vc-test-suite-implementations';
 import {getSuiteConfig} from './test-config.js';
+import {LocalVerifier} from './vc-verifier/index.js';
```

Comment:
usually you only capitalize es6 classes or factory functions. so this should probably be `localVerifier`.